src/Config/FileConfigManager.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace Spatie\Ignition\Config;
+
+use Spatie\Ignition\Contracts\ConfigManager;
+
+class FileConfigManager implements ConfigManager
+{
+    private const SETTINGS_FILE_NAME = '.ignition.json';
+
+    private string $path;
+
+    public function __construct(string $path = '')
+    {
+        $this->path = $this->initPath($path);
+    }
+
+    private function initPath(string $path): string
```

Comment:
For most OSS package we avoid `private` for all properties and functions. This allows the developers to be able to extend and easily override these in their own codebase.

```suggestion
    protected function initPath(string $path): string
```

Reply:
I haven't noticed that, my bad. Thank you for pointing it :)